docs/pages/database-access/guides/sql-server-ad-pkinit.mdx

```diff
@@ -0,0 +1,387 @@
+</Admonition>
+
+### Publish Teleport CA
+
+This step enables the domain controllers to trust the Teleport CA, which will
+allow smart card logons via Teleport to succeed.
+
+On a machine which is joined to your domain and logged in as an account in the
+`Domain Administrators` group, run the two commands below at a PowerShell prompt
+to publish the Teleport CA to your Active Directory domain (using the path
+to the exported Teleport `db-ca.cer` file that you copied above):
+
+```powershell
+certutil –dspublish –f <PathToCertFile.cer> RootCA
+certutil -dspublish -f <PathToCertFile.cert> NTAuthCA
+```
+
+### Publish Teleport CRL
+
+On the same machine, run the command below at a PowerShell promt to publish the
+Teleport CRL to your Active Directory domain (using the path to the exported
+`db-ca.crl` file that you copied above).
+
+```powershell
+certutil -dspublish -f <PathToCRLFile.crl> TeleportDB
+```
+
+<Admonition type="tip">
+  To avoid waiting until the certificate propagation, you can force the CA
+  retrieval from LDAP after importing the CA and CRL with the command:
+
+  ```powershell
+  certutil -pulse
+  ```
+</Admonition>
+
+## Step 3/7. Export LDAP CA certificate
+
+To authenticate users, Teleport uses LDAPS. This means that Teleport needs to
+know that the certificate sent by the server during the initial SSL connection
+is trusted. This is done by specifying the LDAP CA certificate on the database
+configuration, but first, we need to export it from your AD. It can be done by
+running the following PowerShell script on your Windows instance:
+
+```powershell
+$WindowsDERFile = $env:TEMP + "\windows.der"
+$WindowsPEMFile = $env:TEMP + "\windows.pem"
+certutil "-ca.cert" $WindowsDERFile 
+certutil -encode $WindowsDERFile $WindowsPEMFile
+
+$CA_CERT_PEM = Get-Content -Path $WindowsPEMFile
+Write-Output $CA_CERT_PEM
+
+Remove-Item $WindowsDERFile -Recurse
+Remove-Item $WindowsPEMFile -Recurse
+```
+
+The script will write the LDAP CA contents in PEM format to the terminal, and
+from there, you can copy and use it on your database configuration.
+
+## Step 4/7. Set up the Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+<Admonition type="note">
+  Teleport Database Service must run on a Linux server with the `kinit` command
+  installed with PKINIT extensions.
+
+  <Tabs>
+    <TabItem label="Ubuntu">
+      ```code
+      $ sudo apt-get update
+      $ sudo apt-get -y install krb5-user krb5-pkinit
+      ```
+    </TabItem>
+    <TabItem label="RHEL / CentOS 7">
+      ```code
+      $ sudo yum -y update
+      $ sudo yum -y install krb5-workstation krb5-pkinit
+      ```
+    </TabItem>
+  </Tabs>
+</Admonition>
+
+Copy the join token to a file on the instance where you will run the Database
+Service, and then use the following configuration:
```

Comment:
We should be explicit about which fields in the example configuration we expect users to edit.